v23.1/admission-control.md

```diff
@@ -44,7 +44,7 @@ Admission control is enabled  by default for all layers.
 
 When admission control is enabled, request and response operations are sorted into work queues where the operations are organized by priority and transaction start time.
 
-Higher priority operations are processed first. The criteria for determining higher and lower priority operations is different at each processing layer, and is determined by the CPU and storage I/O of the operation. Write operations in the [KV storage layer](architecture/storage-layer.html) in particular are often the cause of performance bottlenecks, and admission control prevents [the Pebble storage engine](cockroach-start.html#storage-engine) from experiencing high read amplification. Critical cluster operations like node heartbeats are processed as high priority, as are transactions that hold locks in order to avoid [contention](performance-recipes.html#transaction-contention) by releasing locks.
+Higher priority operations are processed first. The criteria for determining higher and lower priority operations is different at each processing layer, and is determined by the CPU and storage I/O of the operation. Write operations in the [KV storage layer](architecture/storage-layer.html) in particular are often the cause of performance bottlenecks, and admission control prevents [the Pebble storage engine](architecture/storage-layer.html#pebble) from experiencing high [read amplification](architecture/storage-layer.html#read-amplification). Critical cluster operations like node heartbeats are processed as high priority, as are transactions that hold locks in order to avoid [contention](performance-recipes.html#transaction-contention) by releasing [locks](architecture/transaction-layer.html#writing).
```

Comment:
s/"by releasing locks"/"and release locks earlier".

Reply:
Updated (in both places) to say "and release locks earlier".  Also the 2nd mention of locks is no longer a link, as in my previous comment.